config/locales/server.en.yml

```diff
@@ -39,9 +39,7 @@ en:
       headlines: Headlines
       mark_as_read: Mark as read
       thanks_for_reading: Thanks for reading, have a great day!
-      click_here_to_mark_all_as_read_html: |
-        Here's what happened on Loomio yesterday.
-        Once you've read it, click <a href="%{link}">mark as read</a> to remove it from your Loomio Inbox.
+        heres_what_happened: Here’s what happened on Loomio yesterday.
```

Comment:
Hm, alright. Well the build is erroring because of a parse error in that
file, so there's something up with that change.

On Fri, Feb 26, 2016 at 9:41 AM, mhjb <notifications@github.com> wrote:

> In config/locales/server.en.yml
> <https://github.com/loomio/loomio/pull/3017#discussion_r54160770>:
>
> > @@ -39,9 +39,7 @@ en:
> >        headlines: Headlines
> >        mark_as_read: Mark as read
> >        thanks_for_reading: Thanks for reading, have a great day!
> > -      click_here_to_mark_all_as_read_html: |
> > -        Here's what happened on Loomio yesterday.
> > -        Once you've read it, click <a href="%{link}">mark as read</a> to remove it from your Loomio Inbox.
> > +        heres_what_happened: Here’s what happened on Loomio yesterday.
>
> I'm confused — other lines in that file have curly apostrophes without
> being wrapped in double quotes…
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3017/files#r54160770>.
>